docs/src/tutorials/supported_operations.md

```diff
@@ -0,0 +1,98 @@
+# thin-edge.io Supported Operations
+
+## Supported Operations concepts
+
+### Device operations
+
+IoT devices often do more than just send data to the cloud. They also do things like:
+
+* receive triggers from the operator
+* reboot on demand
+* install or remove software
+
+These are all operations that are supported by Cumulocity IoT and other cloud providers.
+In thin-edge.io we support these operations by writing a plugin that implements the
+[`Operation`](https://cumulocity.com/guides/reference/operation) interface and registering it with the
+thin-edge.io Supported Operations API.
```

Comment:
This introduction can be improved. Notably, it seems better to link the Cumulocity operation doc to Cumulocity  rather than to thin-edge. By the way, the link is broken.

```suggestion
These operations that are supported by [Cumulocity IoT](https://cumulocity.com/api/10.11.0/#section/Device-management-library) and other cloud providers. On thin-edge.io the support for one such operation can be added using the thin-edge.io Supported Operations API.
```